index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atleti</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="//https://fonts.googleapis.com/css2?family=Outfit" />
+</head>
+<header class="main-header">
+
+    <div class="logo">
+        <img src="atletilogo.png" width="200" height="200" alt="AtletiLOGO">
+ 
+        <div class="sitename">
```

Comment:
This is very minor, but I think best practice is separating words with a hyphen when naming classes, to make it easier to read.